packages/cli/test/integration/generators/relation.has-many.integration.js

```diff
@@ -313,11 +313,7 @@ describe('lb4 relation HasMany', function() {
         relationType: 'hasMany',
         sourceModel: 'CustomerClass',
         destinationModel: 'OrderClass',
-      },
-      {
-        relationType: 'hasMany',
```

Comment:
same here, any reason remove it?

Reply:
This test case is the same as 
```
      {
        relationType: 'hasMany',
        sourceModel: 'Customer',
        destinationModel: 'Order',
      },
```
as a default value case. They are the same except the class names.

Reply:
ah I see, fair enough.